apps/prairielearn/src/pages/home/home.ejs

```diff
@@ -117,6 +117,22 @@
         </div><!-- card -->
         <% } %> <!-- devMode -->
 
+        <% if (locals.admin_institutions && locals.admin_institutions.length > 0) { %>
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">Institutions with admin access</div>
+
+            <ul class="list-group list-group-flush">
+              <% locals.admin_institutions.forEach(function(institution) { %>
+                <li class="list-group-item">
+                  <a href="/pl/institution/<%= institution.id %>/admin/courses">
+                    <%= institution.short_name %>: <%= institution.long_name %>
+                  </a>
+                </li>
+              <% }); %>
+            </ul>
+          </div>
+        <% } %>
```

Comment:
In earlier versions, there wasn't any way to get to the institution admin page short of typing in the URL. This fixes that.

Reply:
Haha, I thought that was deliberate, but I'm glad to see it fixed!